frontend/providers/devbox/services/monitorFetch.ts

```diff
@@ -0,0 +1,25 @@
+import { AxiosRequestConfig } from 'axios'
+
+export const monitorFetch = async (props: AxiosRequestConfig, kubeconfig: string) => {
+  const { url, params } = props
+  const queryString = typeof params === 'object' ? new URLSearchParams(params).toString() : params
+  const requestOptions = {
+    method: 'GET',
+    headers: {
+      Authorization: encodeURIComponent(kubeconfig)
+    }
+  }
+
+  const doMain = process.env.MONITOR_URL || 'http://launchpad-monitor.sealos.svc.cluster.local:8428'
```

Comment:
Hard-coded SVC url?

Reply:
The second part is the case that cannot be reached, I will delete it.